selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -368,7 +368,7 @@ void WifiManager::updateGsmSettings(bool roaming, QString apn, bool metered) {
       changes = true;
     }
 
-    int meteredInt = metered ? NM_METERED_NO : NM_METERED_UNKNOWN;
+    int meteredInt = metered ? NM_METERED_UNKNOWN : NM_METERED_NO;
```

Comment:
is `NM_METERED_UNKNOWN` the default for new connections (lets NM decide)?

Reply:
Looks like it https://developer-old.gnome.org/libnm/stable/libnm-nm-dbus-interface.html